backend/src/main/java/hu/flowacademy/worksheet/service/UserService.java

```diff
@@ -31,6 +33,18 @@ public User saveUser(@NonNull User user) throws ValidationException {
         return userRepository.save(user);
     }
 
+    public List<User>  getActiveUsers(@NonNull boolean active) throws ValidationException {
```

Comment:
Legyünk konzisztensek, legyen ez is wrapper class primitív helyett. És ne legyen két space a függvény neve előtt.